web/js/map/ui.js

```diff
@@ -958,10 +986,44 @@ export default function mapui(models, config, store, ui) {
         $zoomOut.button('enable');
       }
     };
-    map.getView().on('change:resolution', lodashDebounce(debouncedZoomChange, 30));
+    const eventListenerKey = map.getView().on('change:resolution', lodashDebounce(debouncedZoomChange, 30));
+    self.zoomButtonListeners.push(eventListenerKey);
     onZoomChange();
   }
-
+  /*
+   * @method mapHasZoomButtons
+   * @static
+   *
+   * @returns {void}
+   */
+  function mapHasZoomButtons(proj) {
+    const zoomInButton = document.querySelector(`#wv-map-${proj} .wv-map-zoom-in`);
+    const zoomOutButton = document.querySelector(`#wv-map-${proj} .wv-map-zoom-out`);
+    return zoomInButton && zoomOutButton;
+  }
+  /*
+   * @method removeZoomButtons
+   * @static
+   *
+   * @returns {void}
+   */
+  function removeZoomButtons() {
```

Comment:
Is there a reason we can't just use CSS media queries to hide them at mobile browser widths?

Reply:
My thought process was to not add if `isMobile` and if a resize changed to `isMobile` then removing the buttons also gets rid of event listeners associated with the zoom buttons.

Reply:
Hmmm makes sense.  I am lazy and would have done the 4 line CSS solution:
```
@media (max-width: 740px) {
  .wv-map-zoom-in, .wv-map-zoom-out {
    display: none;
  }
}
```

... but testing each solution they look/feel exactly the same from a user POV so, I think this is fine as is!

Reply:
I would also lean towards what @jasontk19 stated above (to avoid more mapUI code, and css before js is a general rule of thumb for me)  or just add a mobile class to the outmost div and not worry about the add/remove controls.